src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -310,11 +317,11 @@ private static int LineBreaksBeforeCloseBrace(SyntaxToken closeBraceToken)
             }
         }
 
-        private static int LineBreaksAfterOpenBrace(SyntaxToken currentToken, SyntaxToken nextToken)
+        private static int LineBreaksAfterOpenBrace(SyntaxToken openBraceToken)
```

Comment:
nit: consider adding an assertion that `openBraceToken` is indeed an open brace. Same for `LineBreaksAfterCloseBrace`, `LineBreaksAfterSemicolon`,